part07-CiscoDevNet-provider/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "device_address" {
+  type = string
+}
+variable "username" {
+  type = string
+}
+variable "password" {
+  type = string
+}
+variable "ntp_address" {
+  type = string
+}
+variable "ntp_key" {
+  type = string
+}
+variable "local_subnet" {
+  type = string
+}
+variable "local_subnet_gw" {
+  type = string
+}
```

Comment:
need enter on this line

Reply:
Please update description for all variable , you can pickup all of them from terraform documentation.

Reply:
Done, please review.